docker-compose-auditdb.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.8'
+
+services:
+  mongodb:
+    image: mongo:5.0 #this version should align with available aws document-db versions for image: mongo:5.0
+    container_name: mongodb-audit-db
+    environment:
+      MONGO_INITDB_DATABASE: mongodb-audit-db
+      MONGO_URI: mongodb://localhost:27017/mongodb-audit-db
+    ports:
+      - 27017:27017
+    volumes:
+      - ./data:/data/db
```

Comment:
we should put this at `./data/audit-db:/data/db` so it doesn't conflict with the other postgres data.